sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -216,6 +216,29 @@ export interface CallbackFunctionArgs<E, R> extends BaseCallbackFunctionArgs {
      * will call into each time the Lambda is invoked.
      */
     callbackFactory?: CallbackFactory<E, R>;
+
+    /**
+     * This specifies if the serverless function requires full access to AWS services. This access level would
+     * be similar to the (now deprecated) AWSLambdaFullAccess policy. This would create a copy of that AWS IAM Policy
+     * that would include access to the following services:
+     * - cloudformation
+     * - cloudwatch
+     * - cognito-identity
+     * - cognito-sync
+     * - dynambodb
+     * - ec2
+     * - events
+     * - iam
+     * - iot
+     * - kinesis
+     * - lambda
+     * - logs
+     * - s3
+     * - sns
+     * - sqs
+     * - xray
+     */
+    needsServicesFullAccess?: boolean;
```

Comment:
I can't say I'm a huge fan of this name, but I also don't know if I have any better suggestions. One thought would be to name this more closely to the original intent, so something like `includeDeprecatedLambdaFullAccessPolicy` to be more descriptive?

Reply:
Happy to make this change!